client/src/components/subscription/subscriptionsList.js

```diff
@@ -34,15 +36,37 @@ class SubscriptionsList extends React.Component {
       toast.error('Error: Error getting your subscriptions!');
       return;
     }
+    const subscriptions = await data.json();
+
+    for (let i = 0; i < subscriptions.length; i += 1) {
+      const subscription = subscriptions[i];
+      const { dueDate, name } = subscription;
+
+      if (dueDate.lateDueDate) {
+        toast.error(`Your subscription ${name} was due ${new Date(dueDate.lateDueDate).toLocaleDateString()}`, {
```

Comment:
let's change the order of text so that it says `Your ${sub_name} subscription was due on ${date}` to match the "due today" toast noti

<img width="334" alt="Screen Shot 2022-02-26 at 4 03 07 PM" src="https://user-images.githubusercontent.com/25065329/155858980-2cf260ca-317d-4e5a-bcef-00d1539ff21f.png">